qc/qc_adae.R

```diff
@@ -0,0 +1,20 @@
+###########################################################################
+#' developers : Thomas Neitmann/
+#' date: 16DEC2022
+#' QC ADSL
```

Comment:
QC ADAE

Reply:
Thanks for catching. Updated.

Reply:
@SHAESEN2 or @thomas-neitmann If ok, please `Resolve conversation`.